### ISSUE TITLE
Skipping css file

### ISSUE THREAD

**Issue by eirikurn** [rthor/cra-generate#24]
For js-in-css projects, we don't want a css file for components. Would be nice to have a project setting to skip that, like we can already skip the tests.

**Comment by rthor** [comment]
Good point! I'm wondering if the correct way to go about this would be to rename the `"cssExtension"` property to `"css"` and accept `false` as a value, in addition to css extension strings. Eg

```ts
interface OptionsT {
  [otherOptions: string]: any,
  css: false | string,
}
```

This way we support both and could pass correct overrides in the cli: 
```fish
cra-generate Foo --css=false
```

**Comment by eirikurn** [comment]
I like that.
